orchagent/portsorch.cpp

```diff
@@ -2133,65 +2136,111 @@ void PortsOrch::doPortTask(Consumer &consumer)
                 }
-
                 /* Set port serdes ipredriver */
-                if (fvField(i) == "ipredriver")
+                else if (fvField(i) == "ipredriver")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_IPREDRIVER, attr_val));
+                }
+                /* Set port serdes pre1 */
+                else if (fvField(i) == "pre1")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_PRE1, attr_val));
+                }
+                /* Set port serdes pre2 */
+                else if (fvField(i) == "pre2")
                 {
-                    getPortSerdesVal(fvValue(i), ipredriver);
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_PRE2, attr_val));
+                }
+                /* Set port serdes pre3 */
+                else if (fvField(i) == "pre3")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_PRE3, attr_val));
+                }
+                /* Set port serdes main */
+                else if (fvField(i) == "main")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_MAIN, attr_val));
+                }
+                /* Set port serdes post1 */
+                else if (fvField(i) == "post1")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_POST1, attr_val));
+                }
+                /* Set port serdes post2 */
+                else if (fvField(i) == "post2")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_POST2, attr_val));
+                }
+                /* Set port serdes post3 */
+                else if (fvField(i) == "post3")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_POST3, attr_val));
+                }
+                /* Set port serdes attn */
+                else if (fvField(i) == "attn")
+                {
+                    getPortSerdesVal(fvValue(i), attr_val);
+                    serdes_attr.insert(serdes_attr_pair(SAI_PORT_SERDES_ATTR_TX_FIR_ATTN, attr_val));
+                }
+                else
+                {
+                    SWSS_LOG_ERROR("Unknown port field: %s", fvField(i).c_str());
```

Comment:
The CRM tests are failing b/c the syslog is getting throttled due to this message:
```
Feb  2 23:46:10.029687 9a649eaefbe1 ERR #orchagent: message repeated 4793 times: [ :- doPortTask: Unknown port field: alias]
```

Do we need to add/skip "alias" here?

Reply:
I can remove this as there might be other fields that could be added in the future which may not be significant for the SAI conversion.